lib/parser.js

```diff
@@ -59,15 +59,14 @@ class Parser extends EventEmitter {
       entry.name = Object.keys(entry.value)[0]
       entry.value = entry.value[entry.name]
 
-      if (entry.value instanceof utils.NodeFunction) {
-        switch (property.key.name) {
-          case 'computed':
-            entry.dependencies = utils.getDependencies(entry.value, this.source.script)
-            break
-
-          default:
-            entry.params = entry.value.params
+      if (property.key.name === 'computed') {
+        if (entry.value instanceof utils.NodeFunction) {
+          entry.dependencies = utils.getDependencies(entry.value, this.source.script)
+        } else if (entry.value instanceof Object && entry.value.get instanceof utils.NodeFunction) {
```

Comment:
This branch need to be cover by unit tests

Reply:
I added the unit tests, coverage is back up :-)

Reply:
Good. Now you have to add a unit test to cover the case where we have a missing `get()` function. Some think like:

```js
it('should parse computed prop with missing get() method', (done) => {
  const filename = './fixtures/checkbox.vue'
  const script = `
    export default {
      computed: {
        idGetter: {
          foo () {
            const value = this.value
            return this.name + value
          }
        }
      }
    }
  `
  const options = {
    source: { script },
    filename
  }
  const parser = new Parser(options)

  parser.walk().on('computed', (prop) => {
    if (prop.dependencies.length) {
      return done(new Error('Should parse computed prop without dependencies'))
    }
    done()
  })
})
```